src/packages/@workspaces/features/collection-list/components/request/Request.svelte

```diff
@@ -121,6 +121,19 @@
       inputField.blur();
     }
   };
+
+  function dragStart(event, collection) {
+    const data = {
+      workspaceId: collection.workspaceId,
+      collectionId: collection.id,
+      folderId: folder?.id ?? "",
+      requestId: api.id,
+      name: api.name,
+      method: api.request.method,
+      collection: collection,
```

Comment:
from here we don't need to send the collection, it's already available in testflowexplorer